src/Minimap.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const MINIMAP_WIDTH = 350;
+const MINIMAP_HEIGHT = 250;
```

Comment:
Do we need to control the height and width based on viewport?

Reply:
No, but I made 2 variables incase we wanted to change the width and height of the minimap.

Reply:
make height and width same as reactflow miniMap

Reply:
done